course.py

```diff
@@ -65,6 +65,11 @@ def assign_teachers_to_course(course_id, course_year, assigned_teachers):
         raise e
 
 
+def count_org(orgs):
+    if len(orgs) == 0:
+        return "Please select at least one organization", 400
```

Comment:
Maybe you can use flash cards as you did for other error messages, so that we still land on a usable page once the error is triggered. 

There is no need to return a HTTP 400 code. As long as your form allows the user to submit such content, the server should not return any error code. However, the user should indeed be warned of the missing field.

Reply:
Yes its better my bad